src/stylesheets/dsdl/tokens.css

```diff
@@ -49,7 +50,30 @@
   --dsdl-{{ color.group }}-{{ color.level }}: {{ color.value }};
   {% endfor %}
 
-  --calitp-brand-blue: var(--dsdl-blue-70);  /* #045b86 */
-  --calitp-brand-cyan: var(--dsdl-cyan-40);  /* #38abd0 */
+  --calitp-brand-blue:   var(--dsdl-blue-70);    /* #045b86 */
+  --calitp-brand-cyan:   var(--dsdl-cyan-40);    /* #38abd0 */
   --calitp-brand-yellow: var(--dsdl-yellow-30);  /* #febd28 */
 }
+
+
+/* SPACING */
+
+:root {
+  --spacing-1px: 1px;  /* Non-relative unit for when we don't want a single pixel stroke to scale */
+
+  --spacing-base: 8;  /* Calculated pixel sizes based on this base →→→→→→→→→→→→→ ↓↓↓↓↓ */
+  --spacing-05:   calc(var(--spacing-base) * 0.5 / var(--text-base) * 1rem);  /* 4px   */
```

Comment:
i could be missing something but defining a `px` base and calculating final values in `rem` makes these conversions feel unnecessarily convoluted.

```css
// alternatives
:root {
  --spacing-base: 8px;
  --spacing-05: calc(var(--spacing-base) * 0.5); /* 4px */

  --spacing-base: 0.5rem;
  --spacing-05: calc(var(--spacing-base) * 0.5); /* 4px */
}
```

Reply:
I agree it's convoluted.

The first alternative doesn't work if we want the result units to be `rem`.

The second alternative I don't like exactly as written because when we base things off of pixel values (as in the text sizing above), I like seeing that number. It could be added via comment, though, OR we could do something like this:

```css
:root {
  --spacing-base: calc(8 / var(--text-base) * 1em; /* 8px → 0.5rem */
  --spacing-05: calc(var(--spacing-base) * 0.5); /* 0.25rem (4px) */
}
```

Reply:
Hrmmm. I wonder if that's just how Chrome displays it? I suppose the true test would be does it scale if you change the default text size in the browser prefs. I can double-check this.

Reply:
i also learned that github.com fails to honor non-default browser text sizes completely. i was surprised by this because they've made loads of accessibility improvements over the last few years.